src/utils.jl

```diff
@@ -0,0 +1,10 @@
+function contains_path_traversal(url)
```

Comment:
```suggestion
contains_path_traversal(url::URI) = contains_path_traversal(url.path)
function contains_path_traversal(url::AbstractString)
```

Reply:
Should we have a method that works on URIs themselves? Because working on string conversion of URIs may not be suitable always.